frontend_tests/node_tests/dispatch.js

```diff
@@ -948,6 +948,7 @@ with_overrides(function (override) {
     };
 
     dispatch(event);
+    delete window.electron_bridge; // otherwise this leaks to other tests!
```

Comment:
Eww.  Maybe we should instead add some logic in `zjsunit` to run this command after each test?

Reply:
That sound way better. I moved this to `zjsunit` side of code.